wolfcrypt/src/asn.c

```diff
@@ -6535,9 +6536,17 @@ int GetName(DecodedCert* cert, int nameType, int maxIdx)
 #if (defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)) && \
             !defined(WOLFCRYPT_ONLY)
     if (nameType == ISSUER) {
+#if (defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX)) && defined(WOLFSSL_CERT_EXT)
```

Comment:
I think this macro guard and the one below might not be totally right. See my previous comment where `HAVE_PKCS7` needed to be checked.

Reply:
Fixed.

Reply:
I don't see `HAVE_PKCS7` here. Perhaps the fix was left out of the latest push by mistake?

Reply:
Yup. Put it back in.